src/Symfony/Component/Console/Event/ConsoleCommandEvent.php

```diff
@@ -24,6 +27,11 @@ class ConsoleCommandEvent extends ConsoleEvent
     const RETURN_CODE_DISABLED = 113;
 
     /**
+     * @internal
```

Comment:
fixed. also made the decorator implements `InputInterface` to avoid BC break regarding the return type of `getInput()`.